Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.messages.in

```diff
@@ -139,4 +139,6 @@ messages -> NetworkConnectionToWebProcess LegacyReceiver {
     ClearFrameLoadRecordsForStorageAccess(WebCore::FrameIdentifier frameID)
 
     LoadImageForDecoding(WebCore::ResourceRequest request, WebKit::WebPageProxyIdentifier pageID, size_t maximumBytesFromNetwork) -> (std::variant<WebCore::ResourceError, Ref<WebCore::FragmentedSharedBuffer>> result)
+
+    SetLoginStatus(WebCore::RegistrableDomain domain, enum:uint8_t WebCore::IsLoggedIn loggedInStatus) -> ()
```

Comment:
We should probably disable this IPC endpoint if the login status API is not enabled. For an example, see mentions of `[EnabledIf='isWebTransportEnabled()']` in this file which does this for web transport.